<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="4936" started-at="2019-02-28T11:46:16Z" finished-at="2019-02-28T11:46:21Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="4936" started-at="2019-02-28T11:46:16Z" finished-at="2019-02-28T11:46:21Z">
      <class name="tests.NewEmailTest">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:tests.NewEmailTest@6767c1fc]" name="beforeClass" is-config="true" duration-ms="10" started-at="2019-02-28T17:16:16Z" finished-at="2019-02-28T17:16:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="myTest()[pri:0, instance:tests.NewEmailTest@6767c1fc]" name="myTest" duration-ms="13" started-at="2019-02-28T17:16:16Z" finished-at="2019-02-28T17:16:16Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [80] but found [10]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [80] but found [10]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at tests.NewEmailTest.myTest(NewEmailTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myTest -->
        <test-method status="PASS" signature="afterClass(org.testng.ITestResult)[pri:0, instance:tests.NewEmailTest@6767c1fc]" name="afterClass" is-config="true" duration-ms="4879" started-at="2019-02-28T17:16:16Z" finished-at="2019-02-28T17:16:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=myTest status=FAILURE method=NewEmailTest.myTest()[pri:0, instance:tests.NewEmailTest@6767c1fc] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- tests.NewEmailTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
